config/config_test.go

```diff
@@ -579,7 +579,7 @@ func TestFullConfig(t *testing.T) {
 	cmpStrings(t, "request_validation.ipv6_private_networks", cfg.RequestValidation.IPv6PrivateNetworks[1], "2222::/16")
 	cmpBools(t, "generate_bid_id", cfg.GenerateBidID, true)
 	cmpStrings(t, "debug.override_token", cfg.Debug.OverrideToken, "")
-	cmpBools(t, "enable_legacy_auction", cfg.EnableLegacyAuction, false)
+	cmpBools(t, "enable_legacy_auction", cfg.EnableLegacyAuction, true)
```

Comment:
For the `TestFullConfig` test, can we leave its value as `false` just for the sake of it being different to the default value of `true`?